tests/src/test/scala/org/http4s/headers/CookieSuite.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2013-2020 http4s.org
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.http4s
+package headers
+
+class CookieSuite extends Http4sSuite {
```

Comment:
The old spec was broken:

- No longer supported is a trailing semi-colon.  Did people really need this?
- Arbitrary whitespace around the semi-colon is forbidden.  We could maybe be more forgiving here.
- The RFC is clear that quoted values can't drop the quotes.  This might break apps?